apps/ledger-live-mobile/src/screens/Onboarding/steps/postWelcomeSelection.tsx

```diff
@@ -47,6 +50,10 @@ type PostWelcomeDiscoverCardProps = {
     light: ImageSourcePropType;
     dark: ImageSourcePropType;
   };
+  imageContainerProps?: React.ComponentProps<
+    typeof ChoiceCard
+  >["imageContainerProps"];
+  imageProps?: Partial<React.ComponentProps<typeof Illustration>>;
```

Comment:
same for `IllustrationProps` ? It works, but it's maybe clearer to avoid `React.ComponentProps<typeof>` stuff - WDYT ?